Core/Core/People/PeopleListViewController.swift

```diff
@@ -171,11 +229,7 @@ class PeopleListCell: UITableViewCell {
         avatarView.url = user?.avatarURL
         nameLabel.text = user?.name
         let roles = user?.enrollments?.compactMap { $0.formattedRole }.sorted() ?? []
-        if #available(iOS 13, *) {
-            rolesLabel.text = ListFormatter.localizedString(from: roles)
-        } else {
-            rolesLabel.text = roles.joined(separator: ", ")
-        }
+        rolesLabel.text = ListFormatter.localizedString(from: roles)
```

Comment:
This doesn't require iOS 13 after all, huh?

Reply:
We have our own `ListFormatter`, and I modified it to use the `Foundation` one when available.